tests/monitoring/test_mean_aggregator.py

```diff
@@ -0,0 +1,40 @@
+import theano
+import numpy
+from numpy.testing import assert_allclose
+
+from collections import OrderedDict
+from fuel.datasets import IndexableDataset
+from fuel.streams import DataStream
+from fuel.schemes import SequentialScheme
+from theano import tensor
+
+from blocks.monitoring.evaluators import DatasetEvaluator
+from blocks.monitoring.aggregation import Mean
+
+def test_mean_aggregator():
+  num_examples = 4
+  num_batches = 2
+  features = numpy.array([[0, 3], 
+                       [2, 9],
+                       [2, 4],
+                       [5, 1]], dtype = theano.config.floatX)
```

Comment:
I guess Scrutinizer will complain here about indentation and that it should be `dtype=theano.conf...`, without spaces.